docs/sections/negotiation_guide.ipynb

```diff
@@ -0,0 +1,465 @@
+    "we use the `\"tac_demand\"` data model.  \n",
+    "\n",
+    "The attribute names `tac_good_X_pbk` is the name given to each tradable good.  \n",
+    "Notice that the keys are automatically generated, depending on the number of goods in the game.  \n",
+    "\n",
+    "Depending on the value of the flag `is_supply`, the generated description contains different quantities for each good:\n",
+    "\n",
+    "- If `is_supply` is `True`, then the quantities good are generated by the method `Strategy.supplied_good_quantities(current_holdings)`\n",
+    " and have to be interpreted as the amount of each good the agent is willing to sell;\n",
+    "- If `is_supply` is `False`, then the quantities good are generated by the method `Strategy.demanded_good_quantities(current_holdings)` \n",
+    "and have to be interpreted as the amount of each good the agent is willing to buy; \n",
+    "\n",
+    "Notice that `supplied_good_quantities` and `demanded_good_quantities` are user-defined method to be implemented \n",
+    "in the `Strategy` object, which defines the agent's behaviour.\n",
+    "\n",
+    "Here you can see the output of `BaselineStrategy.supplied_good_quantities` \n",
+    "and `BaselineStrategy.demanded_good_quantities`\n"
+   ],
+   "metadata": {
+    "collapsed": false,
+    "pycharm": {
+     "name": "#%% md\n"
+    }
+   }
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "outputs": [
+    {
+     "name": "stdout",
+     "text": [
+      "[1, 1, 1, 1]\n[1, 2, 3, 0]\n"
+     ],
+     "output_type": "stream"
+    }
+   ],
+   "source": [
+    "from tac.agents.v1.examples.strategy import BaselineStrategy\n",
+    "baseline_strategy = BaselineStrategy()\n",
+    "\n",
+    "current_holdings = [2, 3, 4, 1]\n",
+    "\n",
+    "supplied_good_quantities = baseline_strategy.supplied_good_quantities(current_holdings)\n",
+    "demanded_good_quantities = baseline_strategy.demanded_good_quantities(current_holdings)\n",
+    "\n",
+    "print(\"Supplied quantities: \", supplied_good_quantities)\n",
+    "print(\"Demanded quantities: \", demanded_good_quantities)\n"
+   ],
+   "metadata": {
+    "collapsed": false,
+    "pycharm": {
+     "name": "#%% \n",
+     "is_executing": false
+    }
+   }
+  },
+  {
+   "cell_type": "markdown",
+   "source": [
+    "\n",
+    "The baseline supplied quantities are the current holdings minus `1`. This is because\n",
+    "the first quantity is the most valuable one in terms of utility, \n",
+    "due to the logarithmic shape of the [Cobb-Douglas utility function](https://en.wikipedia.org/wiki/Cobb%E2%80%93Douglas_production_function#Cobb%E2%80%93Douglas_utilities)\n",
+    "\n",
+    "The baseline demanded quantities are just `1` for every good. this is because every good instance is going to be\n",
+    "profitable, due to the ever-increasing utility function.\n",
```

Comment:
Suggest to change:
..is going to be profitable,
>>
..is going to be providing additional utility to the agent,